src/modules/createTask.js

```diff
@@ -0,0 +1,10 @@
+class NewTask {
+  constructor(description, index) {
+    this.description = description;
+    this.index = index;
+    this.id = Math.random().toString(36).substr(2, 7).replace(/[^a-z]/g, '');
```

Comment:
- [ ] Good to implement in the constructor but if you use **index** instead of your **id** code should be clean.

Reply:
Yes, that's right. I used them formerly when I was using classes to filter my tasks. now they're no longer needed

Reply:
Good Job Really happy